website/src/components/pages/single-component/component-page-header-image.js

```diff
@@ -0,0 +1,91 @@
+/** @jsx jsx */
+import { jsx } from '@westpac/core';
+import React from 'react';
+import HeaderImageRight from '../../symbols/WbcBackgroundRightSvg';
+import HeaderImageLeft from '../../symbols/WbcBackgroundLeftSvg';
+import StgHeaderBackground from '../../symbols/StgBackgroundSvg';
+import BsaHeaderBackgroundSvg from '../../symbols/BsaBackgroundSvg';
```

Comment:
You did ask about where to put these. I would handle them the same as any other component. Can just be in the `website/src/components` folder... Maybe to make it more obvious we can stick them into `website/src/components/header`?